docs/sources/tempo/configuration/_index.md

```diff
@@ -1591,6 +1591,11 @@ overrides:
       # This is to filter out spans that are outdated.
       [ingestion_time_range_slack: <duration>]
 
+      # Configures the histogram implementation to use for span-metrics and
+      # service-graphs processors.  If native histograms are desired, the
+      # receiver must be configured to ingest native histograms.
```

Comment:
I haven't used hyphens when referring to span-metrics and service-graph in the docs before. Is this usage specific for something in the code or are you using them as compound adjectives?

Reply:
I'd like to align here, so I should use `servicegraphs` or `service graphs`?  I see some `service_graphs` for the YAML key, but not much else here.

Reply:
I've dropped the hyphens after taking a look around.